tutorials/intermediate/arranging_axes.py

```diff
@@ -0,0 +1,368 @@
+"""
+===================================
+Arranging multiple Axes in a Figure
+===================================
+
+Often more than one "Axes" is wanted on a figure at a time, usually
+organized into a regular grid.  Matplotlib has a variety of tools for
+working with grids of Axes that have evolved over the history of the library.
+Here we will discuss the tools we think users should use most often, the tools
+that underpin how Axes are organized, and mention some of the older tools.
+
+.. note::
+
+    We use "Axes" to refer to the object that contains plotting
```

Comment:
```suggestion
    We use *Axes* to refer to the object that contains plotting
```
IMHO italics are more suited to mark a technical term.

Reply:
OK, so I changed this throughout.

Reply:
Oh, I'd only suggest to do it for the first usage in the definition.

E.g. https://apastyle.apa.org/style-grammar-guidelines/italics-quotations/italics

> When to use italics - First use of key terms or phrases, often accompanied by a definition

Reply:
OK< fixed, I think